src/AddTodoForm.js

```diff
@@ -1,11 +1,30 @@
-import React from "react";
+import React, {useState} from "react";
+
+
+let AddTodoForm = ({onAddTodo}) => {
+
+  const[inputText, setInputText] = useState("");
+  
+
+  let handleAddTodo = function (event){
+    event.preventDefault(); // Prevent the default behaviour of the form submit.
+
+    // Retrieves the value of the title element from 
+    // the evenet target and store it in a variable named todoTitle.
+    let todoTitle = inputText; 
+    console.log(todoTitle);
+    setInputText(event.target.value);
```

Comment:
You're so close to getting the value! In fact, I might have seen you correctly get it a commented out line. It'll be event.target.title.value.

Reply:
Okay.

Thank you.

I will follow your recommendations and check if the app runs as expected.

I sincerely appreciate your efforts and time Mark.

Thank you so much.